var/spack/repos/builtin/packages/synapsetool/package.py

```diff
@@ -34,6 +34,7 @@ class Synapsetool(CMakePackage):
     git      = "ssh://bbpcode.epfl.ch/hpc/synapse-tool"
 
     version('develop', submodules=True)
+    version('0.5.4', tag='v0.5.3', submodules=True)
```

Comment:
typo on tag?

Reply:
No, that's intended. This is WIP, triggering a rebuild for debugging purpose.